File: fontend/lib/domain/dtos/create_todo_dto.dart
class CreateTodoDTO {
  final String title;
  final DateTime executeDay;
  final DateTime notificationTime;
  final DateTime eventDay;

  CreateTodoDTO({
    required this.title,
    required this.executeDay,
    required this.notificationTime,
    required this.eventDay,
  });
}

File: backend/src/todos/dtos/create-todo.dto.ts
import { ApiProperty } from '@nestjs/swagger';
import { IsDateString, IsString } from 'class-validator';

export class CreateTodoDTO {
  @ApiProperty()
  @IsString()
  title: string;

  @ApiProperty()
  @IsDateString()
  execute_day: Date;

  @ApiProperty()
  @IsDateString()
  event_day: Date;

  @ApiProperty()
  @IsDateString()
  notification_time: Date;
}
